Sources/Widgets/Widgets3D/SeedWidget/state.js

```diff
@@ -0,0 +1,18 @@
+import vtkStateBuilder from 'vtk.js/Sources/Widgets/Core/StateBuilder';
+
+// Defines the structure of the widget state.
+// See https://kitware.github.io/vtk-js/docs/concepts_widgets.html.
+export default function stateGenerator() {
+  return vtkStateBuilder
+    .createBuilder()
+    .addStateFromMixin({
+      labels: ['moveHandle'],
+      mixins: ['origin', 'color3', 'scale1', 'visible', 'manipulator'],
```

Comment:
a seed could also have a 'direction' mixin

Reply:
Ok, changed in [9e9d8e8](https://github.com/Kitware/vtk-js/pull/2907/commits/9e9d8e87f4dafe68e5aa259594c118b150c599b3)

Reply:
After adding the mixin, I tried out the example again. The cube no longer appears as expected: 
![image](https://github.com/Kitware/vtk-js/assets/44912689/c4a4dc68-6349-40f1-ab3c-f9081d1b866f)

If I comment out the direction mixin, it looks good again: 
![image](https://github.com/Kitware/vtk-js/assets/44912689/8637e9d0-5e43-42a2-bc4a-fa008ad3ca68)

Is there something more I need to do to make the direction mixin work properly?

Reply:
With the `direction` mixin added, the widget still appears as a 2D gray/black rectangle. Is the mixin compatible? How can I fix the appearance?

Reply:
What if you set the initial direction value to something else like `[0,0,1]`?

Reply:
Thanks, that fixed it! The cubes now look normal: 
![image](https://github.com/Kitware/vtk-js/assets/44912689/88a35192-4a98-4354-bd30-c59871c56a83)

Reply:
Huh, not sure why the initial [1,0,0] breaks rendering.